tests/unit/conftest.py

```diff
@@ -39,6 +39,11 @@ def LOCATION():
     yield "us-central"
 
 
+@pytest.fixture
+def PYARROW_MINIMUM_VERSION():  # pragma: NO COVER
+    return "3.0.0"
+
+
```

Comment:
Doesn't look like this fixture is used?

Also if it is used, could this use the constant from `_version_helpers` as the tests do below?